src/include/sof/lib/pm_runtime.h

```diff
@@ -44,7 +44,8 @@ enum pm_runtime_context {
 	SSP_CLK,			/**< SSP Clock */
 	DMIC_CLK,			/**< DMIC Clock */
 	DMIC_POW,			/**< DMIC Power */
-	DW_DMAC_CLK			/**< DW DMAC Clock */
+	DW_DMAC_CLK,			/**< DW DMAC Clock */
+	CORE_MEMORY_POW			/**< Core Memory power */
 };
```

Comment:
Add comma after last element to simplify the diff when the next element will be added. 

``CORE_MEMORY_POW,``